docs/dev/framework/models/_index.md

```diff
@@ -25,8 +25,8 @@ Each class extending `Model` has some *static* methods you can use:
 ```php
 use Contao\PageModel;
 
-// get a record by its primary key
-$page = PageModel::findById(2);
+// get a record by its primary key (usually the ID)
+$page = PageModel::findByPk(2);
```

Comment:
The `Model::findByPk()` method is deprecated and should not be used anymore - thus this needs to be reverted.

Reply:
Shouldn't we then also remove the method elsewhere in this document (or at least add some info about the deprecation)?

Reply:
Also, since when is this deprecated? I thought only the use with null is deprecated?

Reply:
Yes, all `::findByPK` references need to be replaced by `::findById` - which I have done now in 09c0fb2
I don't think we need to add a deprecation info about it in the docs.

Reply:
Okay, reverted it.